sso-kit-core/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1 @@
+com.vaadin.sso.core.SingleSignOnDefaultBeans
```

Comment:
`SingleSignOnDefaultBeans` is now in the two `starter`s

Reply:
Yes, only starters should define beans. The core module is not a starter, so it should not define any beans.

Reply:
Oh now I see what you mean! I'll fix it 😅

Reply:
Fixed, thanks 🙏